applications/rag/README.md

```diff
@@ -106,19 +126,19 @@ EOF
 * At the end of the smoke test with the TGI server, close the port forward for 8080.
```

Comment:
nit: add steps for how to close port forward

Reply:
Easy with two terminals. Done.